app/src/main/res/values-si/string.xml

```diff
@@ -0,0 +1,1027 @@
+
+    <string name="device_not_connected">උපාංගය සම්බන්ධ කර නොමැත</string>
+    <string name="use_autoscan">PSLab උපාංගය සමඟ සම්බන්ධ වී ඇති සංවේදක හඳුනා ගැනීමට ස්වයං-සුපරීක්ෂක බොත්තම ඔබන්න </string>
+    <string name="sensor_not_connected">I2C මඟින් සංවේදක කිසිවක් සම්බන්ධ කර නොමැත. පහත ලැයිස්තුවෙන් සංවේදකයක් තෝරාගෙන අන්තර්-සංගෘහිත පරිපථ (I2C) බසය ක්‍රියාකරන ආකාරය අධ්‍යනය කරන්න. සම්බන්ධ කිරීමෙන් පසු ස්වයං-සුපරීක්ෂක බොත්තම ඔබන්න</string>
+    <string name="scanning">සුපිරීක්ෂණය …</string>
+
+    <string name="voltage_unit">(V)</string>
+    <string name="volts">වෝල්ට්</string>
+    <string name="seconds">(s)</string>
+    <string name="lux">Lx</string>
+    <string name="pressure2">පීඩන.</string>
+    <string name="plot_altitude">සලකුණ - අක්ෂාංශ</string>
+    <string name="celcius">(°C)</string>
+    <string name="plot_temperature">සලකුණ - උශ්ණත්වය</string>
+    <string name="guass">Guass</string>
+    <string name="guass_unit">(Ga)</string>
+    <string name="humid">ආර්ද්‍රතාවය</string>
+    <string name="humid_unit">(RH)</string>
+    <string name="plot_humidity">සලකුණ - ආර්ද්‍රතාවය</string>
+    <string name="illum">Illum.</string>
+    <string name="lux2">(Lux)</string>
+    <string name="acceleration">ත්වරණය.</string>
+    <string name="acceleration_unit">(m/s²)</string>
+    <string name="range">පරාසය</string>
+    <string name="wave1">තරංගය 1</string>
+    <string name="wave2">තරංගය 2</string>
+
+    <string name="start_logging">ලඝු කිරීම ආරම්භ කරන්න</string>
+    <string name="stop_logging">ලඝු කිරීම අවසන් කරන්න</string>
+    <string name="stop">නවතන්න</string>
+    <string name="cancel">අවලංගු කරන්න</string>
+    <string name="ok">හරි</string>
+    <string name="save_data">දත්ත සුරකින්න</string>
+    <string name="view_logged_data">ලඝු කරන ලද දත්ත දර්ශනය කරන්න</string>
+    <string name="export_logged_data">>ලඝු කරන ලද නිර්යාත කරන්න</string>
+    <string name="sensor_logged_data">ලඝු වූ සංවේදක දත්ත</string>
+
+    <string name="full">පිරී අවසන්</string>
+
+    <string name="pocket_science_logo">Pocket Science ලෝගෝව</string>
+
+    <string name="timebase_microsec">%1$.2f μs</string>
+    <string name="timebase_milisec">%1$.2f ms</string>
+    <string name="time_space">කාලය </string>
+    <string name="unit_μs">(μs)</string>
+    <string name="label_ch1">CH1</string>
+    <string name="label_V">(V)</string>
+    <string name="label_ch2">CH2</string>
+    <string name="channel_parameters">නාලිකා පරාමිතියන්</string>
+    <string name="timebase_and_trigger">කාලරාමුව සහ ප්‍රේරකය</string>
+    <string name="data_analysis">දත්ත විශ්ලේෂණය</string>
+    <string name="xy_plot">XY Plot</string>
+    <string name="hundred_ms">100ms</string>
+    <string name="hyphen">-</string>
+    <string name="plus">+</string>
+    <string name="select_a_value">අගයක් තෝරන්න</string>
+    <string name="ch3_value">CH3 (+/- %1$.1fV)</string>
+    <string name="one_value">1.0</string>
+    <string name="fourier_transforms">ෆූරියර් ට්‍රාන්ස්ෆෝමර්</string>
```

Comment:
ෆූරියර් පරිණාමන

Reply:
@Piumal1999 it should be ෆූරියර් පරිණාමක

Reply:
පරිණාමක means transformers @jayasanka-sack

Reply:
ah yes!